communities/microgalaxy/lab/templates/intro.html

```diff
@@ -28,13 +28,9 @@ <h5 class="modal-title">Galaxy {{ site_name }}</h5>
           <p>
             This site <em>{{ subdomain }}.{{ root_domain }}</em> is connected to the same server as <em>{{ root_domain }}</em>, but with an interface dedicated to helping our {{ analysis_name }} researchers. Your history, jobs and data quota are shared with the "Main" Galaxy {{ site_name }}  website.
           </p>
-          <div class="my-3">
-            Switch between sites quickly using the dropdown in Galaxy {{ site_name }}'s main navigation bar:
-            <img class="w-100 my-3 mx-auto" src="https://github.com/usegalaxy-au/galaxy-media-site/blob/dev/webapp/home/static/home/img/subdomains/site-switcher.png?raw=true" alt="Switch site button">
-          </div>
-          <p><a class="ga-btn" href="{{ galaxy_base_url }}" target="_blank">Take me back to Galaxy {{ site_name }}</a></p>
-          <p><a class="ga-btn" href="/about">What is Galaxy {{ site_name }}?</a></p>
-          <p><a class="ga-btn" href="/request/support">Galaxy {{ site_name }} support</a></p>
+          <p><a class="ga-btn" href="{{ root_domain }}" target="_blank">Take me back to Galaxy {{ site_name }}</a></p>
+          <p><a class="ga-btn" href="https://usegalaxy.eu/about">What is Galaxy {{ site_name }}?</a></p>
```

Comment:
This URL will depend on the site. It could be added as a variable (e.g. `doc`) in the https://github.com/galaxyproject/galaxy_codex/blob/main/communities/microgalaxy/lab/usegalaxy.eu.yml files

Reply:
I changed that to the root domain too